docs/entity_creation.md

```diff
@@ -0,0 +1,207 @@
+# Entity Creation
+
+
+
+Expanding the current selection of indexed entities is a way of increasing the coverage of queryable network interactions. 
+
+### Implementation
+
+
+#### Adding the entity schema
+
+in `schema.graphql`:
+
+The GraphQl schema is amended to include the new entity, with any fields that are needed to support new use cases. These attributes are preferably only those required in order to minimise storage.
+```
+0  type TestEntity @entity {
+1  		id: ID!
+2   	entity_field_a: String! @index
+3   	entity_field_b: String! @index
+4   	entity_field_c: String!
+5   	message: Message!
+6   	transaction: Transaction!
+7   	block: Block!
+8  }
+```
+
+- L0 - Entity is defined with the `@entity` annotation
```

Comment:
```suggestion
- L0 - Entity is defined as a GraphQL type with the addition of the `@entity` annotation
```

Reply:
It looks like some of these lines have periods and others don't. Let's try to be consistent with our formatting to aid readability. I don't have a strong opinion either way (yes/no periods), I would defer to what you think looks best.